SendWithUs.Client/Requests/ISendRequest.cs

```diff
@@ -108,6 +113,12 @@ public interface ISendRequest : IRequest
         string TemplateVersionId { get; }
 
         /// <summary>
+        /// Gets the language tag for a localized variant of the template identified by TemplateId.
+        /// </summary>
+        /// <remarks>Corresponds to the "locale" property in the SendWithUs API.</remarks>
+        string Language { get; }

```

Comment:
Still don't understand the choice to rename this to Language rather than keep the name as Locale. You explained that it's a language code, sure, but that's what a ton of APIs use to specify a locale. Was reading [this spec](http://www.w3.org/International/core/langtags/#locale-lang-standards) which reads:

    Locales can be identified in several ways. One method is by inference from language tags. 


Changing the name just intros a barrier to people who might be already familiar with or are reading the SendWithUs docs.